resources/assets/js/components/CreateMember.js

```diff
@@ -55,13 +56,15 @@ class CreateMember extends Component {
   handleSubmit(e){
     e.preventDefault();
     const formData = new FormData();
+    this.state.memberDob = this.state.memberDob ? this.state.memberDob : $('#date_of_birth').val();
     formData.append('name',this.state.memberName);
     formData.append('information',this.state.memberInformation);
     formData.append('avatar',this.state.memberAvatar);
     formData.append('dob',this.state.memberDob);
     formData.append('phone',this.state.memberPhone);
     formData.append('position',this.state.memberPosition);
     formData.append('gender',this.state.memberGender);
+    formData.append('MAX_FILE_SIZE',20907152);
```

Comment:
Here you are using a max size of about 20MB, but the requirement states it should be 10MB max.

Reply:
i have do it